<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OWIS_controller</class>
 <widget class="QMainWindow" name="OWIS_controller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OWIS_controller</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="InitPS90Button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Init PS90</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop all axes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InitStageButton_X">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Init X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InitStageButton_Y">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Init Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InitStageButton_Z">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Init Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_stage_state_X">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>60</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X Stage state</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_stage_state_Y">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>110</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Stage state</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_stage_state_Z">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>160</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z Stage state</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShutOffStage_X">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Shut off X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShutOffStage_Y">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Shut off Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShutOffStage_Z">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Shut off Z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DisconnectButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect PS90</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_PS90_general_status">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>General Status PS90</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShutOnStage_X">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Shut on X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShutOnStage_Y">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Shut on Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShutOnStage_Z">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Shut on Z</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>941</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Movement</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_axis_pos_X">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>240</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>New Z target: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_axis_movement_X">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X Axis movement state</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_axis_pos_Z">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y Axis position</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_axis_movement_Y">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y Axis movement state</string>
      </property>
     </widget>
     <widget class="QPushButton" name="homeButton_Z">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>250</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Home Z</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_axis_pos_Y">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_axis_target_mode_X">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Target mode :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="moveAxisButton_Z">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>250</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Move Z stage</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_newTarget_value_Y">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>170</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_axis_target_mode_Z">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Target mode :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="switchModeButton_Z">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>290</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Switch Mode</string>
      </property>
     </widget>
     <widget class="QPushButton" name="switchModeButton_X">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Switch Mode</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X Axis position</string>
      </property>
     </widget>
     <widget class="QPushButton" name="moveAxisButton_Y">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>160</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Move Y stage</string>
      </property>
     </widget>
     <widget class="QPushButton" name="homeButton_Y">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>160</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Home Y</string>
      </property>
     </widget>
     <widget class="QPushButton" name="switchModeButton_Y">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>200</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Switch Mode</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>New Y target: </string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_newTarget_value_X">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="moveAxisButton_X">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>70</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Move X stage</string>
      </property>
     </widget>
     <widget class="QPushButton" name="homeButton_X">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>70</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Home X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_axis_target_mode_Y">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Target mode :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>New X target: </string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_newTarget_value_Z">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>260</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Z Axis position</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_axis_movement_Z">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>290</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Z Axis movement state</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Joystick</string>
     </attribute>
     <widget class="QPushButton" name="joyNegButton_X">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>- X</string>
      </property>
     </widget>
     <widget class="QPushButton" name="joyPosButton_X">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>+ X</string>
      </property>
     </widget>
     <widget class="QPushButton" name="joyNegButton_Y">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>- Y</string>
      </property>
     </widget>
     <widget class="QPushButton" name="joyPosButton_Y">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>+ Y</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
